manifests/c/CocCoc/CocCoc/120.0.6099.234/CocCoc.CocCoc.locale.vi-VN.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: CocCoc.CocCoc
+PackageVersion: 120.0.6099.234
+PackageLocale: vi-VN
+Publisher: Công ty TNHH Cốc Cốc
+PackageName: Cốc Cốc
+License: Proprietary
+Copyright: Copyright 2018 Google LLC
+ShortDescription: Cốc Cốc Browser is a freeware web browser developed by the Vietnamese company Cốc Cốc. It is based on the Chromium open-source code and is specifically tailored for the Vietnamese market.

```

```suggestion
ShortDescription: Trình duyệt Cốc Cốc là trình duyệt web phần mềm miễn phí được phát triển bởi công ty Cốc Cốc của Việt Nam. Nó dựa trên mã nguồn mở Chrome và được thiết kế riêng cho thị trường Việt Nam.
```
(Internet translation - feel free to improve.)

The `ShortDescription` should be in the localization file's language. Could this be translated?

---

yes

---

the translation is correct, resolving this conversation